evmcore/tx_pool.go

```diff
@@ -465,22 +465,23 @@ func (pool *TxPool) SubscribeNewTxsNotify(ch chan<- NewTxsNotify) notify.Subscri
 	return pool.scope.Track(pool.txFeed.Subscribe(ch))
 }
 
-// GasPrice returns the current gas price enforced by the transaction pool.
-func (pool *TxPool) GasPrice() *big.Int {
+// MinTip returns the current gas price enforced by the transaction pool.
+func (pool *TxPool) MinTip() *big.Int {
 	pool.mu.RLock()
 	defer pool.mu.RUnlock()
 
-	return new(big.Int).Set(pool.gasPrice)
+	return new(big.Int).Set(pool.minTip)
 }
 
-// SetGasPrice updates the minimum price required by the transaction pool for a
-// new transaction, and drops all transactions below this threshold.
-func (pool *TxPool) SetGasPrice(price *big.Int) {
+// setMinTip is a test function that updates the minimum tip required by the
```

Comment:
nit: what is a "test function" in this context? Is it a function intended to be used in tests? Please clarify.

If it is a function only supposed to be used in tests, maybe it should be moved to a `*_test.go` file.

Reply:
function moved to `tx_pool_test.go` as a standalone